nyc.config.js

```diff
@@ -2,9 +2,9 @@
 
 const path = require('path');
 module.exports = {
-  all: true,
```

Comment:
this is not strictly a part of scope, but a fix I encountered the need for in working.
and it says "based on all files" not simply the ones touched by tests, and opened up a possibility of inconsistency between coverage measured local vs in bamboo, most notably because nyc produces javascript files if you ask it to produce coverage report in html, which can get included under "all"